UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -33,6 +35,15 @@ public override void OnStartClient()
 			ClientEnsureCorrectState();
 		}
 
+		[ClientRpc]
+		private void AutoCorrect()
+		{
+			//(Max): This is a quick bandage for the ping issue related to laydown behaviors on the clients getting desynced at high pings.
+			//It's still unclear why the state gets desynced at high pings, but this should stop bodies from standing up whenever
+			//they're affected by wind or are thrown around while laying down.
```

Comment:
surely be better to wrap, the stuff that toggles the laying down state as server only?

Reply:
but it is?